anta/input_models/connectivity.py

```diff
@@ -23,13 +23,15 @@ class Host(BaseModel):
     source: IPv4Address | IPv6Address | Interface
     """Source address IP or egress interface to use."""
     vrf: str = "default"
-    """VRF context. Defaults to `default`."""
+    """VRF context."""
     repeat: int = 2
-    """Number of ping repetition. Defaults to 2."""
+    """Number of ping repetition."""
     size: int = 100
-    """Specify datagram size. Defaults to 100."""
+    """Specify datagram size."""
     df_bit: bool = False
-    """Enable do not fragment bit in IP header. Defaults to False."""
+    """Enable do not fragment bit in IP header."""
+    expected_unreachable: bool = False
```

Comment:
@gmuloc Do you prefer the following?
```suggestion
    expected_reachable: bool = True
```

Reply:
I think I prefer 

```
reachable: bool = True
```

Reply:
> I think I prefer
> 
> ```
> reachable: bool = True
> ```

I agree with this suggestion.

Reply:
UPDATED. THANKS!!